arboretum/kubernetes/reports/compliance_oscal_observations.py

```diff
@@ -0,0 +1,227 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2020 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+The compliance OSCAL observations report.
+
+A json report comprising NIST OSCAL Assessment Results Observations generated
+by processing compliance operator fetcher cluster_resource evidence. The
+embedded XML within the cluster_resource evidence is transformed to produce the
+report. If an optional oscal_metadata file is specified, then the report is
+enhanced accordingly.
+
+Provide the "start" and "end" optional configuration (--config) parameters
+as a JSON string, in "YYYYMMDD" format to define a date range for the evidence
+used to process the report.  If omitted, the default value is the current date.
+
+---------------
+Example usages:
+---------------
+
+> harvest report my-repo arboretum compliance_oscal_observations
+
+> harvest report my-repo arboretum compliance_oscal_observations \
+--config '{ \
+"oscal_metadata":"raw/kubernetes/oscal_metadata.yaml" \
+}'
+
+> harvest report my-repo arboretum compliance_oscal_observations \
+--config '{ \
+"cluster_resource":"raw/kubernetes/cluster_resource.json", \
+"oscal_metadata":"raw/kubernetes/oscal_metadata.yaml", \
+"start":"20200901", \
+"end":"20201231" \
+}'
+
+--------------------
+oscal_metadata.yaml:
+--------------------
+
+The oscal_metadata.yaml file comprises one or more mappings. Below is shown the
+format of a single mapping. The items in angle brackets are to be replaced with
+desired values for augmenting the produced OSCAL.
+
+The mapping whose <name> matches the [metadata][name] in the evidence for the
```

Comment:
This is confusing.  Above you imply that this yaml is evidence as is the cluster resources evidence.  If that's the case then you should be clear which evidence you're referring to here.

Reply:
Changed to specify "cluster resources evidence" as suggested.